var/spack/repos/builtin/packages/tecplot/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Tecplot(Package):
+    """Tecplot 360 is a Computational Fluid Dynamics (CFD) and numerical
+    simulation software package used in post-processing simulation results.
+    It is also used in chemistry applications to visualize molecule structure
+    by post-processing charge density data."""
+
+    homepage = "https://www.tecplot.com/"
+    version('2017r1', '06a8057d33a519607720d4c621cd3f50', expand=False)
+    version('2018r2', 'd3cf54a7555e0259b7ba0d82fef23bc3', expand=False)
+
+    def url_for_version(self, version):
+        return "file://{}/tecplot360ex{}_linux64.sh".format(os.getcwd(), version)
```

Comment:
```suggestion
        return "file://{0}/tecplot360ex{1}_linux64.sh".format(os.getcwd(), version)
```
Needed for Python 2.6 support.